### PR TITLE
Enable cluster provisioning on all e2e

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

**Comment by chxchx** [comment]
/hold

bookinfo and simple tests are not required at the moment by I want to see them passing before the merge bot acting too fast. Anyone please comment hold cancel once they pass

**Comment by sebastienvas** [comment]
it should work however I discovered that when we do the test command for a given target, since we have 2 entries in the prow config, it triggers both and since we have the same name, the second starting abort the first one which is why it failed when I manually try to start it. I ll check with the k8s guys if they ever encountered this.

File: prow/e2e-suite.sh [hunk 1]
```diff
@@ -30,11 +30,27 @@ set -u
...
 set -x
 
-TESTS_TARGETS="e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade"
+TEST_TARGETS=(e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade)
 SINGLE_MODE=false
-E2E_ARGS=()
 
-source ${ROOT}/prow/lib.sh
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
+# for exiting resources types
```
Comment (rkpagadala) [inline prow/e2e-suite.sh]: existing

File: prow/istio-pilot-e2e.sh [hunk 2]
```diff
@@ -29,14 +29,27 @@ set -u
 # Print commands
 set -x
 
-source ${ROOT}/prow/lib.sh
-setup_and_export_git_sha
-
-export NUM_NODES=4
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
+# for exiting resources types
```
Comment (rkpagadala) [inline prow/istio-pilot-e2e.sh]: typo: existing

File: prow/istio-pilot-e2e.sh [hunk 3]
```diff
@@ -29,14 +29,27 @@ set -u
 # Print commands
 set -x
 
-source ${ROOT}/prow/lib.sh
-setup_and_export_git_sha
-
-export NUM_NODES=4
```
Comment (rkpagadala) [inline prow/istio-pilot-e2e.sh]: Is this not needed? How do we get 4 nodes for this test?

File: prow/istio-pilot-e2e.sh [hunk 4]
```diff
@@ -29,14 +29,27 @@ set -u
...
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
```
Comment (rkpagadala) [inline prow/istio-pilot-e2e.sh]: the url is wrong

File: prow/e2e-suite.sh [hunk 5]
```diff
@@ -30,11 +30,27 @@ set -u
 # Print commands
 set -x
 
-TESTS_TARGETS="e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade"
+TEST_TARGETS=(e2e_simple e2e_mixer e2e_bookinfo e2e_upgrade)
 SINGLE_MODE=false
-E2E_ARGS=()
 
-source ${ROOT}/prow/lib.sh
+# Check https://github.com/istio/test-infra/blob/master/boskos/configs.yaml
```
Comment (rkpagadala) [inline prow/e2e-suite.sh]: url is wrong

File: prow/istio-pilot-e2e.sh (hunk 4 above)
Comment (sebastienvas) [reply prow/istio-pilot-e2e.sh]: Yes the PR is still pending review.

File: prow/e2e-suite.sh (hunk 5 above)
Comment (sebastienvas) [reply prow/e2e-suite.sh]: Yes the PR is still pending review.

Comment (sebastienvas) [reply prow/e2e-suite.sh]: PR is pending. https://github.com/istio/test-infra/pull/690

File: prow/e2e-suite.sh (hunk 1 above)
Comment (sebastienvas) [reply prow/e2e-suite.sh]: done

File: prow/istio-pilot-e2e.sh (hunk 4 above)
Comment (sebastienvas) [reply prow/istio-pilot-e2e.sh]: PR pending

File: prow/istio-pilot-e2e.sh (hunk 2 above)
Comment (sebastienvas) [reply prow/istio-pilot-e2e.sh]: done

File: prow/istio-pilot-e2e.sh (hunk 3 above)
Comment (sebastienvas) [reply prow/istio-pilot-e2e.sh]: the pool already created cluster with 4 nodes. https://github.com/sebastienvas/test-infra/blob/eba53851a4e94bda1abb2b1e50e5ba0457a750d7/boskos/configs.yaml#L12

**Pull request merged**